### DISCUSSION TITLE
Organisation

### DISCUSSION THREAD

**Discussion by MagicLike** [JulianPrieber/littlelink-custom#160] [Feature requests]
What about moving the whole project with @JulianPrieber 's themes and stuff to an organisation for better overview?

**Comment by MagicLike** [comment]
Also themes from authors without a GitHub account like the Aurora theme could be added there without really "belonging" to anyone...

**Comment by JulianPrieber** [comment]
https://github.com/LittleLink-Custom

**Reply by MagicLike** [reply]
Nice! Can you then please move the repos over and create one for the Aurora theme? (or I can create the repo, but I need to be a member with org-repo rights for that).

**Comment by JulianPrieber** [comment]
I transferred ownership of one of my repositories and the transfer went seamless.
The original URL still works, (which is quite important in this case): https://github.com/JulianPrieber/phone-dial
And the new one obviously works too: https://github.com/littlelink-custom/phone-dial

All the issues stars etc. get transferred over, so I might actually transfer all the main repositories, not sure yet.

**Reply by MagicLike** [reply]
If everything is in fine, I would definitely do it.

**Comment by ghost** [comment]
Maybe to make it easier to find/contribute to themes, you could setup a separate organisation for themes: https://github.com/LittleLink-Themes

I'm involved in the BackdropCMS project, and we do something similar there:
- https://github.com/backdrop/ (core repos)
- https://github.com/backdrop-contrib/ (add-ons, themes, etc.)
